backend/src/main/java/de/neuefische/booklibrary/backend/book/BookController.java

```diff
@@ -0,0 +1,92 @@
+package de.neuefische.booklibrary.backend.book;
+
+import de.neuefische.booklibrary.backend.api.ApiBookService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.validation.Valid;
+import java.util.List;
+
+import static de.neuefische.booklibrary.backend.api.ApiBookService.DEFAULT_COVER;
+
+
+@RestController
+@RequestMapping("/api/books")
+@RequiredArgsConstructor
+public class BookController {
+
+    private final ApiBookService apiBookService;
+    private final BookService bookService;
+
+    @GetMapping
+    List<Book> getAllBooks() {
+        return bookService.getAllBooks();
+    }
+
+
+    @GetMapping("/search/{searchText}")
+    public List<Book> getAllApiBooks(@PathVariable String searchText) {
+        return apiBookService.getAllApiBooks(searchText);
+    }
+
+    @GetMapping("/isbn/{isbn}")
+    public List<Book> getApiBookByIsbn(@PathVariable @Valid String isbn) {
```

Comment:
`@Valid` wird bei `String` keinen Effekt haben (wenn ich nichts übersehe).
```suggestion
    public List<Book> getApiBookByIsbn(@PathVariable String isbn) {
```

Reply:
ja, die annotation ist hier überflüssig